networking_oneview/ml2/drivers/oneview/common.py

```diff
@@ -152,19 +154,62 @@ def get_uplinkset_by_name_in_lig(oneview_client, lig_id, uplinkset_name):
     return uplinkset
 
 
-def get_logical_interconnect_group_by_id(oneview_client, lig_id):
+def get_logical_interconnect_group_by_id(lig_id):
     """Get a Logical Interconnect Group Object to a given LIG id.
 
-    :param oneview_client: a instance of the OneView Client;
     :param lig_id: the id of the Logical Interconnect Group;
     :returns: the Logical Interconnect Group object
-    :raise HPOneViewException: If was not possible to retrieve LIG;
+    :raise OneViewResourceNotFoundException: If it was not possible
+        to retrieve LIG;
     """
+    oneview_client = get_oneview_client()
```

Comment:
why are you not using the reauth system?

Reply:
It only works for classes that store the oneview_client as attribute.